src/platform/common/platform/fs-paths.ts

```diff
@@ -11,8 +11,8 @@ export function getFilePath(file: Uri | undefined) {
     if (file) {
         const fsPath = uriPath.originalFSPath(file);
 
-        // Remove separator on the front
-        if (fsPath && fsPath.startsWith(path.sep) && isWindows) {
+        // Remove separator on the front if not a network drive
+        if (fsPath && fsPath.startsWith(path.sep) && fsPath.length > 1 && fsPath[1] !== path.sep && isWindows) {
```

Comment:
You might want to upstream this to VS Code.
Do you have an example for what we're trying to fix here.
Would be good to have a sample in the code as comments.

Reply:
Good idea. I'll add examples.